src/transformers/modeling_longformer.py

```diff
@@ -33,6 +33,7 @@
 LONGFORMER_PRETRAINED_MODEL_ARCHIVE_MAP = {
     "longformer-base-4096": "https://s3.amazonaws.com/models.huggingface.co/bert/allenai/longformer-base-4096/pytorch_model.bin",
     "longformer-large-4096": "https://s3.amazonaws.com/models.huggingface.co/bert/allenai/longformer-large-4096/pytorch_model.bin",
+    "longformer-base-4096-finetuned-triviaqa": "https://s3.amazonaws.com/models.huggingface.co/bert/allenai/longformer-base-4096-finetuned-triviaqa/pytorch_model.bin",
```

Comment:
stuck to the naming style we also used in Bert for squad finetuned models - hope that's ok @ibeltagy